src/main/java/com/faforever/client/domain/DivisionBean.java

```diff
@@ -0,0 +1,71 @@
+package com.faforever.client.domain;
+
+import javafx.beans.property.IntegerProperty;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleIntegerProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import lombok.Value;
+
+@EqualsAndHashCode(callSuper = true, onlyExplicitlyIncluded = true)
+@ToString(callSuper = true, onlyExplicitlyIncluded = true)
+@Value
+public class DivisionBean extends AbstractEntityBean<DivisionBean> {
+  StringProperty descriptionKey = new SimpleStringProperty();
+  IntegerProperty index = new SimpleIntegerProperty();
+  @ToString.Include
+  StringProperty nameKey = new SimpleStringProperty();
+  ObjectProperty<LeagueSeasonBean> leagueSeason = new SimpleObjectProperty<>();
```

Comment:
Does this not have a technicalName?

Reply:
yes, it actually doesn't have a technical name